src/_respondTo.js

```diff
@@ -0,0 +1,14 @@
+import { css } from "styled-components";
+import { breakpoints } from "_variables";
+
+export const respondTo = Object.keys(breakpoints).reduce(
```

Comment:
please rename to `respondToBreakpoint`.

`respondTo.js` is a quite confusing file name, and looking at the code inside it's not that easy to understand what does it do.